CHANGELOG.md

```diff
@@ -28,6 +28,15 @@
   ([PR #837](https://github.com/alphagov/govuk-frontend/pull/837) and
    [PR #848](https://github.com/alphagov/govuk-frontend/pull/848))
 
+- Use relative line-height
+  Update typography styles to use relative, unitless line-height
+  ([PR #837](https://github.com/alphagov/govuk-frontend/pull/837))
+
+- Allow og:image meta tag to be customisable
```

Comment:
It was already customisable by modifying `assetPath` – I think we need to be clearer as to what the change is here.